### ISSUE TITLE
please help me out for this error

### ISSUE THREAD

**Issue by piyush05124** [keras-team/keras#14147]
i am getting this error

C:\Windows\system32>python C:\Users\kesha\AppData\Local\Continuum\anaconda3\etc\keras\load_config.py  1>temp.txt

C:\Windows\system32>set /p KERAS_BACKEND= 0<temp.txt

C:\Windows\system32>del temp.txt

C:\Windows\system32>python -c "import keras"  1>nul 2>&1

C:\Windows\system32>if errorlevel 1 (
ver  1>nul
 set "KERAS_BACKEND=theano"
 python -c "import keras"  1>nul 2>&1
)

C:\Windows\system32>SET DISTUTILS_USE_SDK=1

C:\Windows\system32>SET MSSdk=1

C:\Windows\system32>SET platform=

C:\Windows\system32>IF /I [AMD64] == [amd64] set "platform=true"

C:\Windows\system32>IF /I [] == [amd64] set "platform=true"

C:\Windows\system32>if defined platform (set "VSREGKEY=HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\14.0" )  ELSE (set "VSREGKEY=HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\14.0" )

C:\Windows\system32>for /F "skip=2 tokens=2,*" %A in ('reg query "HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\14.0" /v InstallDir') do SET "VSINSTALLDIR=%B"
ERROR: The system was unable to find the specified registry key or value.

C:\Windows\system32>if "" == "" (set "VSINSTALLDIR=" )

C:\Windows\system32>if "" == "" (
ECHO "WARNING: Did not find VS in registry or in VS140COMNTOOLS env var - your compiler may not work"
 GOTO End
)
"WARNING: Did not find VS in registry or in VS140COMNTOOLS env var - your compiler may not work"
The system cannot find the batch label specified - End

**Comment by Chinmay922** [comment]
Questions,
Which IDE are you using? Could you also tell me which version of VS you're using on your machine?

**Comment by tilakrayal** [comment]
@piyush05124 ,
Can you please take a look at this [issue](https://github.com/keras-team/keras/issues/13209) with similar error.It helps.Thanks!

**Comment by piyush05124** [comment]
> @piyush05124 ,
> Can you please take a look at this [issue](https://github.com/keras-team/keras/issues/13209) with similar error.It helps.Thanks!

1.  FOR VC++ 17 AND BELOW IT WORKS WELL BUT ABOVE 17 IT HAS LOTS OF ISSUES.
2. TO TACKLE THIS SCENARIO , SINCE I GENERALLY USE TENSORFLOW, I INSTALLED THE LATEST VERSION OF KERAS AS WELL AS TENSORFLOW AND IT REALLY WORKS. 
3. IN FACT VSCODE HAS SEVERAL OTHER ISSUES ALSO WITH IT. SO BETTER DIRECTLY INSTALL KERAS AND TENSORFLOW INTO YOUR MACHINE AND YOU WILL BE ABLE TO IMPORT IT IN ANY EDITOR UNLESS YOU CREATE A DIFFERENT VIRTUAL ENVIRONMENT.

**Comment by tilakrayal** [comment]
@piyush05124 ,
Glad the issue has resolved in latest versions.Please feel free to close the issue if the issue has resolved.Thanks!
